web/packages/shared/components/FieldSelect/FieldSelectCreatable.tsx

```diff
@@ -67,6 +67,7 @@ export function FieldSelectCreatable({
   elevated = false,
   inputId = 'select',
   markAsError = false,
+  customProps,
```

Comment:
Could you add a JSDoc for `customProps`?

https://github.com/gravitational/teleport/blob/30e0e2f1baf83c8ae3f08999be7aa6b27ecbdcc2/web/packages/shared/components/Select/types.ts#L48